public/app/plugins/datasource/loki/queryHints.ts

```diff
@@ -63,5 +68,30 @@ export function getQueryHints(query: string, series: DataFrame[]): QueryHint[] {
     }
   }
 
+  const queryWithLabelFormat = isQueryWithLabelFormat(query);
+  if (!queryWithLabelFormat) {
+    const hasLevel = dataFrameHasLevelLabel(series[0]);
+    const levelLikeLabel = extractLevelLikeLabelFromDataFrame(series[0]);
+
+    // Add hint only if we don't have "level" label and have level-like label
+    if (!hasLevel && levelLikeLabel) {
+      hints.push({
+        type: 'ADD_LEVEL_LABEL_FORMAT',
+        label: `Some logs in your selected log stream have "${levelLikeLabel}" label.`,
```

Comment:
Maybe this whole copy could be improved. @grafana/docs-squad do you have any suggestions?